src/App.tsx

```diff
@@ -1,25 +1,47 @@
-import React from 'react';
-import logo from './logo.svg';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
 import './App.css';
 
+export interface Breed{
+  name: string
+}
+export interface GetBreedsResponse {
+  data: Breed[]
+}
+
+export const getBreeds = async (): Promise<Breed[]|any>  => {
+  try{
+    return await axios.get('https://api.thecatapi.com/v1/breeds');
```

Comment:
Supondo que essa aplicação vá a produção e vc está chamando uma api propria, a gente não vai querer em ambiente de dev mudar dados em prod, ne? Sugiro configurar as chamadas da api por variavel de ambeinte. Vc pode dar uma olhadinha na lib `dotenv` e/ou criar um arquivo com com valores default.

// ./envVars.ts
```ts
export const CATS_API = "'https://api.thecatapi.com"
```

```suggestion
    return await axios.get(`${CATS_API}/v1/breeds');
```

Reply:
Coloquei a API em uma pasta dela 9c98b8b5831a7f4267af3554677ad2488953a507 , eu tentei usar o env mas não consegui. Vou tentar novamente :)